plugins/Diagnostics/Diagnostic/PageSpeedCheck.php

```diff
@@ -61,7 +61,7 @@ private function isPageSpeedEnabled()
                 $getExtendedInfo = true
             );
         } catch(\Exception $e) {
-            $this->logger->info('Unable to test if mod_pagespeed is enabled: the request to {url} failed', array(
+            $this->logger->info('Unable to test if mod_pagespeed is enabled: the request to {url} failed' . $e->getMessage(), array(
```

Comment:
Better use the [PSR-3 context](http://www.php-fig.org/psr/psr-3/#1.3-context) rather than concatenating